.spelling

```diff
@@ -99,7 +99,9 @@ AssemblyScript
 AttributeGen
 Auth0
 AuthenticationPolicy
+AuthorizationPolicy
 AuthPolicy
+authz
```

Comment:
Remove

Reply:
Removing this will cause spell checking failure

Reply:
Removed